### ISSUE TITLE
Why no jpg?

### ISSUE THREAD

**Issue by spib** [hey-red/MimeTypesMap#2]
Hi,

I'm using this in windows and it seems that .jpg isn't supported, only .jpeg. While I'm sure that .jpeg is the 'correct' extension, would it be possible to also support .jpg?

Thanks!

**Comment by hey-red** [comment]
We can add it like this:

`MimeTypesMap.AddOrUpdate("image/jpeg", "jpg");`

then
```
MimeTypesMap.GetMimeType("test.jpg") //=> image/jpeg
MimeTypesMap.GetMimeType("test.jpeg") //=> image/jpeg
```

**Comment by spib** [comment]
Thanks for the quick reply, it's great that there is the flexibility to do that but I think most users would expect .jpg to work out of the box? 

Is it perhaps worth considering having that included by default?

**Comment by hey-red** [comment]
This is possible, because alternative extensions are present in the apache list, but only the first extension will be added to the dictionary.
I can fix it, but not today(maybe tomorrow).

**Comment by hey-red** [comment]
Fixed 6a88632
Nuget package has been updated.
